config/config.go

```diff
@@ -57,6 +57,7 @@ type Proxy struct {
 	TLSHeader             string
 	TLSHeaderValue        string
 	GZIPContentTypes      *regexp.Regexp
+	UniqueRequestID       string
```

Comment:
I think `RequestID` is enough since they need to be unique for this to make sense.